src/router/response.rs

```diff
@@ -1,14 +1,16 @@
 use super::ResponseBody;
 
 use async_std::fs;
+use cookie::Cookie;
 use http::StatusCode;
 use hyper::{header, Body};
 use serde::ser::Serialize;
 
 pub struct Response {
     body: Body,
     status: StatusCode,
-    headers: Option<Vec<(header::HeaderName, &'static str)>>,
+    headers: Option<Vec<(header::HeaderName, String)>>,
```

Comment:
Why not use `Cow<&'static str>` here to allow both `&str` and `String` with just little computation cost? If we want it to be smaller, we can use https://crates.io/crates/beef.